open_schools_platform/common/filters.py

```diff
@@ -150,6 +151,12 @@ def get_dict_filters_without_prefix(filters):
 
         return response
 
+    def _has_provided_filter(self, filter_type: Type):
+        for name, filter_field in self.get_filters().items():
+            if isinstance(filter_field, filter_type) and self.data.get(name, False):
```

Comment:
why do we need this check?
```self.data.get(name, False)```

Reply:
if no order value was provided, it still would have a filter in self.get_filters